auto-run-nondex/runNondex.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 DIR="${PWD}"
+nondex_version="1.1.2"
+java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*".*/\1\2/p;')
+echo java_version "$java_version"
+if [[ "$java_version" -ge 90 ]]; then
+    echo version is more than 1.9
+    nondex_version="2.1.1"
```

Comment:
Version 2.1.1 supports Java 9+ (at least until 11) *and* Java 8. Why not always run with 2.1.1 instead of 1.1.2?

Reply:
I have completed the modification. As for why I don't always run with 2.1.1 instead of 1.1.2, I was thinking that some projects may require certain features that only 1.1.2 provides. If it weren't for these restrictions, we could certainly run with 2.1.1 directly.

Reply:
I'm not aware of any feature supported by 1.1.2 that's not supported by 2.1.1. Do you have some example? If yes, please share. If not, just change the version in one place below and remove these lines for checking the Java version.